src/scripts/crowdin/import/utils.ts

```diff
@@ -0,0 +1,171 @@
+import { copyFileSync, existsSync, mkdirSync, readdirSync } from "fs"
+import { join } from "path"
+
+import i18Config from "../../../../i18n.config.json"
+import { DOT_CROWDIN } from "../translations/constants"
+
+import { BucketsList, SelectionItem, SummaryItem, TrackerObject } from "./types"
+
+/**
+ * Some language codes used in the repo differ from those used by Crowdin.
+ * This is used to convert any codes that may differ when performing folder lookup.
+ */
+export const getCrowdinCode = (code: string): string =>
+  i18Config.filter((lang) => lang.code === code)?.[0]?.crowdinCode || code
+
+/**
+ * Reads `ls` file contents of `_path`, moving .md and .json files
+ * to their corresponding destinations in the repo. Function is called
+ * again recursively for subdirectories.
+ *
+ * @param _path An absolute path to the directory being scraped.
+ * @param contentSubpath The subpath deep to the lang-code directory,
+ * used to construct destination for markdown content files
+ * @param repoLangCode Language code used within the repo
+ * @returns void
+ */
+export const scrapeDirectory = (
```

Comment:
This function do more than just scrape the dir. We could break out the logic with something like this and simplify the function
```ts
function scrapeDirectory(dir, cb) {
  // ...
  ls.forEach(cb)
}

scrapeDirectory(dir, (path) => {
  // do our logic
})
```

Having done this, we could move `scrapeDirectory` to somewhere in `@/libs/utils` since it is a reusable function.

Reply:
@pettinarip Curious if you see this as blocking? This logic was ported forward from the previous `crowdin-import.ts` file. Any chance we could handle this refactor separately? Similar with above comment about migrating things to `@/lib/utils/crowdin`

Reply:
No, not a blocker, was more an refactor idea to make it clearer. Got it, makes sense what you are saying! we need to investigate a bit about those aliases and ways to use them in scripts.